src/main/java/databaseintegration/DatabaseAppointment.java

```diff
@@ -0,0 +1,41 @@
+package databaseintegration;
+
+import java.sql.*;
+import java.util.ArrayList;
+
+public class DatabaseAppointment {
+    private final Connection connection;
+    private final Statement statement;
+
+    public DatabaseAppointment (Connection connection, Statement statement) {
+        this.connection = connection;
+        this.statement = statement;
+    }
+
+    public void addAppointment (int appointmentID, int clinicID, String clientID, int periodID, int batchID) throws
```

Comment:
If we have time we may want to address the long method header (builder maybe?) but not a priority right now